new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -98,6 +147,12 @@ public void testCreatingDomain() {
     logger.info("Deleted service account \'" + serviceAccount.getMetadata().getName()
         + "' in namespace: " + serviceAccount.getMetadata().getNamespace());
 
+    // Delete the persistent volume claim and persistent volume
+    assertDoesNotThrow(
```

Comment:
Add message string for when it does throw an exception. What should the engineer debugging this look for as the problem?